data/campaigns/Heir_To_The_Throne/scenarios/05b_Isle_of_the_Damned.cfg

```diff
@@ -34,6 +34,95 @@
         [/objectives]
     [/event]
 
+    # This appoints the Advisor role to our most advanced unit coming out of The Bay of Pearls. This would be
+    # the same as the initial Advisor we would appoint if we took the path through Muff Malal's Peninsula, and
+    # will be the initial Advisor recalled at the start of The Siege of Elensefar.
+    #
+    # We create a shallow copy of this Advisor, needing only the name, type and profile image. The shallow copy
```

Comment:
Yes I want to select a specific unit from the recall list. But not to recall because the list is about to be wiped and won't be restored until after I need the message on the screen.

The idea above is: "Act as if there were a unit at this location (47,11), select the tile as if unit were speaking, pop up a message, use this name ("A  hollow voice") and this image ("dark-crossing.png") and say this message ("Plugh!").

In particular, assume we found Symir ... I want the following to work. Not really, but to show the point ...

    [store_unit]
        id=Symir
        x,y=47,11
        variable=symir_store
        kill=yes
    [/store_unit]
    [message]
       speaker=$symir_store.name
       image=$symir_store.profile
       x,y=47,11
       message="Thank god we found you!"
    [/message]

I just stored and killed the unit. Now I want it to speak as if it were at (47,11) which happens to look like a ship.

Or I guess can make a unit of type=ship, name=Symir, located at (47,11), with the profile image of a Knight, I suppose. Then I simply remove the overlay ('ship'), create the unit type=ship and have it speak.

Is there a ship unit type? That might work. Still making a shallow unit but it avoids all that ability mess.

Of course, next time, it will be a windmill which, I'm fairly sure, is not a unit type.